api/server/api-routes.js

```diff
@@ -10,160 +10,110 @@ router.get("/", function (req, res) {
 });
 // Import user controller
 let userController = require("./controllers/userController");
-let groupController = require("./controllers/groupController")
+let groupController = require("./controllers/groupController");
+
 // Import meeting controller
 var meetingController = require("./controllers/meetingController");
 
 // User routes
 
 // Create user
-router
-  .route("/users")
-  .post(userController.register);
+router.route("/user/create").post(userController.register);
+
+//Login user
+router.route("/user/login").post(userController.login);
 
 // Get user by Id
 router
-  .route('/user/:userId')
+  .route("/user/:userId")
   .get(userController.view)
+  .delete(userController.delete)
+  .patch(userController.update);
 
-// Get user groups 
-router
-  .route('/user/groups/:userId')
-  .get(userController.userGroups);
+//Get user groups
+router.route("/user/groups/:userId").get(userController.userGroups);
 
-// Get users by username 
-router
-  .route('/users/:username')
-  .get(userController.usersByUsername)
+//Get users by username
+router.route("/users/:username").get(userController.usersByUsername);
 
-// Add group to list of group requests the user has
+//Add group to list of group requests the user has
 router
-  .route('/user/addGroupRequest')
-  .post(userController.addGroupRequest)
+  .route("/user/addGroupRequest/:userId/:groupId")
+  .post(userController.addGroupRequest);
 
 // Remove group request from the list of group requests the user has
 router
-  .route('/user/removeGroupRequest')
-  .post(userController.removeGroupRequest)
+  .route("/user/removeGroupRequest/:userId/:groupId")
+  .post(userController.removeGroupRequest);
 
 // Add group to the list of user's groups
-router
-  .route('/user/addGroup')
-  .post(userController.addGroup)
+router.route("/user/addGroup/:userId/:groupId").post(userController.addGroup);
 
 // Remove group from the list of user's groups
 router
-  .route('/user/removeGroup')
-  .post(userController.removeGroup)
-
-// Get user meetings
-router
-  .route('/user/meetings/:userId')
-  .get(userController.userMeetings);
-
-// Add meeting to the list of user's meetings
-router
-  .route('/user/addMeeting')
-  .post(userController.addMeeting);
-
-// Remove meeting from the list of user's meetings
-router
-  .route('/user/removeMeeting')
-  .post(userController.removeMeeting);
```

Comment:
I think this is why builds are failing. Because they were removed, the tests that check them would definitely fail

Reply:
How come these were removed? I think we need these routes.

Reply:
removed on accident i think